viewer/packages/camera-manager/src/StationaryCameraManager.ts

```diff
@@ -176,8 +176,34 @@ export class StationaryCameraManager implements CameraManager {
 
   private readonly zoomCamera = (event: WheelEvent) => {
     const sensitivityScaler = 0.05;
+    const preCursorRay = this.getCursorRay(event).normalize();
+    const lastFov = this._camera.fov;
+
     this._camera.fov = Math.min(Math.max(this._camera.fov + event.deltaY * sensitivityScaler, 10), this._defaultFOV);
     this._camera.updateProjectionMatrix();
```

Comment:
Can this also be moved to after the `if`-statement below?

Reply:
The preCursorRay needs to happen before fov is changed, but I moved the check for if the fov has changed further up so that we do no extra work if there is no change in fov. 👍